contracts/nouns/dkg.sol

```diff
@@ -0,0 +1,194 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4;
+
+import "./babyjubjub/CurveBabyJubJub.sol";
+import "../interfaces/IVerifierRound2.sol";
+import "../interfaces/IDkg.sol";
+
+contract DKG is IDkg {
+        uint constant babyjub_sub_order = 2736030358979909402780800718157159386076813972158567259200215660948447373041;
+
+    IVerifierRound2 round2_verifier;
+
+    //
+    // Committee configuration
+    //
+
+    uint public n_comm;
+    uint public tally_threshold;
```

Comment:
Not part of your changes, but while we are touching this code, could we rename this to something generic like `threshold`?

Reply:
Fixed